pkg/util/k8sutil/k8sutil.go

```diff
@@ -89,7 +90,7 @@ func (k K8SOps) EventRecorder() record.EventRecorder {
 // GetDeploymentsUsingStorageClassProvisioner returns a DeploymentList that use a given
 // StorageClass name in the PVC.
 func (k K8SOps) GetDeploymentsUsingStorageClassProvisioner(provisionerName string) (*appsv1.DeploymentList, error) {
-	deployments, err := k.client.AppsV1().Deployments("").List(metav1.ListOptions{})
+	deployments, err := k.client.AppsV1().Deployments("").List(context.TODO(), metav1.ListOptions{})
```

Comment:
Looks like we should accept and pass a context in the k8sutil functions - let's do this separately though.